File: Hour 1/Q2/book_author.java
import java.util.Scanner;
class book_author {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        try{
            String batch = new String();
            batch = sc.next().toLowerCase();
            int i, j, unique = 0, count;
            boolean flag = true;
            for(i = 0; i < batch.length(); i++){
                flag = true;
                count = 0;
                for(j = 0; j < batch.length(); j++){
                    if(batch.charAt(i) == batch.charAt(j))
                        count++;
                    if(count > 1){
                        flag = false;
                        break;
                    }
                }
                if(flag)
                    unique++;
            }
            System.out.println(unique);
        }
        finally{
            sc.close();
        }
    }
}

File: Hour 1/Q2/book_author.c
#include<stdio.h>
#include<ctype.h>
#include<string.h>

int main(){
    char batch[20];
    scanf("%s", batch);
    int i, j, unique = 0, count, flag;
    for(i = 0; i < strlen(batch); i++)
        batch[i] = tolower(batch[i]);
    for(i = 0; i < strlen(batch); i++){
        flag = 1;
        count = 0;
        for(j = 0; j < strlen(batch); j++){
            if(batch[i] == batch[j])
                count++;
            if(count > 1){
                flag = 0;
                break;
            }
        }
        if(flag)
            unique++;
    }
    printf("%d",unique);
    return 0;
}
